### PR TITLE
chore: upgraded ts eslint to version 5.38

### PR DESCRIPTION
**Pull request by ilteoood**
It should resolve #605 and #606

### CODE REVIEW

**Comment by ilteoood** [comment]
It should resolve #605 and #606

**Comment by simoneb** [comment]
> It should resolve #605 and #606

I doubt it, but let's see :)

**Review by simoneb — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -15,7 +15,7 @@
     "db:migrate": "postgrator",
     "db:down": "docker-compose down",
     "lint": "eslint --ext .ts,.js ./src",
-    "test": "cross-env TS_NODE_FILES=1 c8 tap --no-cov --ts"
+    "test": "cross-env TS_NODE_FILES=1 c8 tap -j2 --no-cov --ts"
```
Comment (ilteoood) [inline package.json:18]: It seems that we are doing too much parallel tests, and the CI hasn't enough power to execute both all the tests and the postgres image in under 30 seconds

**Comment by ilteoood** [comment]
I've also refactored the tests to avoid unecessary `await` on `json()` function invocation and to spawn the lowest number of fastify's instances.

**Comment by marco-ippolito** [comment]
I see past ci runs failing, maybe some try launching the ci by sending some dummy commits to see of this solves the issue

**Review by marco-ippolito — APPROVED** [review]

**Review by simoneb — APPROVED** [review]

**Pull request merged**